java/native/HelloWorld/client/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Use the OpenJDK image as the base image
```

Comment:
I'd build it with multistage building.
Final image should use JRE only since app is already compiled. It'll create smaller images.